CONTRIBUTING.md

```diff
@@ -62,3 +63,47 @@ variable:
 ```bash
 $ export EXERCISES_API_URL=http://localhost:9292
 ```
+
+### Track Anatomy
+
+Each track should have the following structure:
+
+```bash
+├── .gitignore
+├── .travis.yml
+├── LICENSE
+├── README.md
+├── SETUP.md
+├── bin
+│   └── fetch-configlet
+├── config.json
+├── docs
+│   ├── ABOUT.md
+│   ├── INSTALLATION.md
+│   ├── LEARNING.md
+│   ├── RESOURCES.md
+│   └── TESTS.md
+└── exercises
+    └── hello_world
```

Comment:
The slug for hello world should be `hello-world` (the website went down the other day because I didn't have enough checks and balances in place to catch this, and the API blew up because there's no such thing as `hello_world`, even though one of the tracks thought so). I've fixed my tests.